unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -31,9 +30,10 @@ public class AvatarShape : BaseComponent
         public bool everythingIsLoaded;
 
         private Vector3? lastAvatarPosition = null;
-        private MinimapMetadata.MinimapUserInfo avatarUserInfo = new MinimapMetadata.MinimapUserInfo();
         bool initializedPosition = false;
 
+        private PlayerStatus playerStatus = null;
```

Comment:
In order to avoid confussions, maybe a better name for this `playerStatus` would be `avatarStatus`? As this data is managed by any avatar in the world and not only by our avatar... What do you think?

Reply:
It contains information about the player, such as name and the voice chat status along the position. I think it would be more confussing to have it named `avatarStatus`.